src/pages/guidelines/2x-grid/implementation.mdx

```diff
@@ -1343,6 +1344,42 @@ Image size can change ratios when the breakpoint changes.
   </DoDont>
 </DoDontRow>
 
+## API
+
+There are many ways to make the 2x Grid work for your application using our API.
+The example below illustrates removing visibility of a given component based on
+the maximum width of the `md` breakpoint. Check out the table to learn more
+about our API.
+
+```css
+.component {
+  display: inline;
+
+  /* applies styles inside mixin on md breakpoint or below, like max-width */
+  @include carbon--breakpoint-down('md') {
+    display: none;
+  }
```

Comment:
The adding of helper classes in https://github.com/carbon-design-system/carbon/pull/11786 was intended to replace the need for this type of manual config. Should we just point them to those classes instead?

Reply:
I think the grid mixins would still be needed for other use cases, no? Like if you wanted to change the layout or styles of something at mobile breakpoints (not necessarily just hide it). @tay1orjones but we should also add those classes and surface them here! I think surfacing all options is good.

Reply:
Great points all around. Let me do another rev where have both options listed, but do a little progressive disclosure to link out to our github docs so things don't get out of sync like Taylor mentions in his other comment.